terraform/outputs.tf

```diff
@@ -4,6 +4,7 @@ output "default_apigatewayv2_stage_invoke_url" {
 
 output "api_gw_api_key" {
   value = aws_api_gateway_api_key.kuberoke.value
+  sensitive = true
```

Comment:
if you want to avoid outputting this in the CLI, I think it's better to remove this entirely from the outputs